mmv1/templates/terraform/examples/dns_domain_tutorial.tf.erb

```diff
@@ -0,0 +1,50 @@
+#[START dns_domain_tutorial]
+# to setup a web-server
+resource "google_compute_instance" "default" {
+  name         = "<%= ctx[:vars]['dns_compute_instance'] %>"
+  machine_type = "g1-small"
+  zone         = "us-central1-b"
+
+  boot_disk {
+    initialize_params {
+      image = "debian-cloud/debian-9"
+    }
+  }
+
+  network_interface {
+    network = "default"
+       access_config {
+           // Ephemeral public IP
+   }
+  }
+  metadata_startup_script = "sudo apt-get update && sudo apt-get install apache2 -y && echo '<!doctype html><html><body><h1>Hello World!</h1>'`date`'</body></html>' > /var/www/html/index.html"
```

Comment:
the `'` characters are throwing off our tests: https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6024-old..auto-pr-6024#diff-40524a01cbca31921bea35a20c5fb30f7fae6a968e81337c5d581d71c1108530R70

since they wrap the entire terraform config in `'` marks, and is causing an error in our test code. Is there another way we could write the startup script, or maybe escape the `'` marks?